Fn ptrs (#100)
RalfJung/minirust: lang/syntax.md

    /// A pointer pointing into a global allocation with a given offset.
    Pointer(Relocation),

> 
>     /// A function pointer value, used for named dynamic pointer
>     FnPtr(FnName),

Reviewer: ```suggestion
    /// A pointer pointing to a function.
    FnPointer(FnName),
```
Might also make sense to rename `Constant::Pointer` to `GlobalPointer`?
